packages/core/src/lib/markdown-it/index.js

```diff
@@ -14,6 +14,7 @@ markdownIt.use(require('markdown-it-mark'))
   .use(require('markdown-it-table-of-contents'))
   .use(require('markdown-it-task-lists'), {enabled: true})
   .use(require('markdown-it-linkify-images'), {imgClass: 'img-fluid'})
+  .use(require('markdown-it-texmath'), {engine: require('katex'), delimiters: 'dollars'})
```

Comment:
quick comment:
let's use `brackets` instead, since we're planning to use `$$` for underline
It should also be the more conventional format I think

Reply:
I'll do that!

Reply:
@ang-zeyu done!